sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_service_client.hpp

```diff
@@ -179,6 +179,43 @@ namespace Azure { namespace Storage { namespace Blobs {
         const std::string& tagFilterSqlExpression,
         const FindBlobsByTagsSinglePageOptions& options = FindBlobsByTagsSinglePageOptions()) const;
 
+    /**
+     * @brief Creates a new blob container under the specified account. If the container with the
+     * same name already exists, the operation fails.
+     *
+     * @param blobContainerName The name of the container to create.
+     * @param options Optional parameters to execute this function.
+     * @return A BlobContainerClient referencing the newly created container.
+     */
+    Azure::Core::Response<BlobContainerClient> CreateBlobContainer(
+        const std::string& blobContainerName,
+        const CreateBlobContainerOptions& options = CreateBlobContainerOptions()) const;
+
+    /**
+     * @brief Marks the specified blob container for deletion. The container and any blobs
+     * contained within it are later deleted during garbage collection.
+     * 
+     * @param blobContainerName The name of the container to delete.
+     * @param options Optional parameters to execute this function.
+     * @return Nothing.
+     */
+    Azure::Core::Response<std::nullptr_t> DeleteBlobContainer(
```

Comment:
Why are we having an inconsistent return type here?

Reply:
explained in this issue https://github.com/Azure/azure-sdk-for-cpp/issues/1322

Reply:
I created a PR #1331 to implement `Response<void>`. If core team accepts this PR, I will change the return type here.

Reply:
duplicate